bedrock/privacy/templates/privacy/privacy-day.html

```diff
@@ -1,142 +1,310 @@
 {# This Source Code Form is subject to the terms of the Mozilla Public
  # License, v. 2.0. If a copy of the MPL was not distributed with this
- # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 
 {% extends "base-resp.html" %}
 
-{% block body_id %}privacy-day{% endblock %}
+{% from "macros.html" import share_cta with context %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{ _('Get smart on privacy') }}{% endblock %}
 
-{% block page_title %}{{ _('We put your privacy first') }}{% endblock %}
+{% block page_image %}{{ media('img/privacy/you/sharing-tile.png') }}{% endblock %}
+
+{% block page_desc %}
+  {{ _('Feel like you’ve lost control of how your information is used online?') }}
+  {{ _('Here are 4 easy steps to raising your privacy IQ & protecting yourself online.') }}
+{% endblock %}
 
-{% block extrahead %}
+{% block body_id %}privacy-day{% endblock %}
+
+{% block site_css %}
   {{ css('privacy-day') }}
 {% endblock %}
 
-{% block body_class %}privacy-day{% endblock %}
+{%
+set share_urls = {
+    'twitter': 'http://mzl.la/1E63Mxm',
+    'googleplus': 'https://www.mozilla.org/' + LANG + '/privacy/tips/?utm_source=Google&utm_medium=MozLandingPage&utm_campaign=privacy2015&utm_content=WebsiteShare',
+    'facebook': 'https://www.mozilla.org/' + LANG + '/privacy/tips/?utm_source=Facebook&utm_medium=MozLandingPage&utm_campaign=privacy2015&utm_content=WebsiteShare#play'
```

Comment:
Is the `#play` hash needed for sharing this page, or is this unintentionally left over from the independence campaign link?